content/subscription/build-cloud/manage-seats.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Manage seats and invites
+description: Learn how to add and remove seats for your Docker Build Cloud subscription, and invite members to your team.
+keywords: subscription, team, business, features, build, cloud, Build Cloud, remote builder, manage users, add seats, remove seats
+---
+
+You can add seats and manage invitations to your Docker Build Cloud Team in the Docker Build Cloud dashboard.
+
+> **Note**
+>
+>If you have a Docker Build Cloud Business subscription, you can add and remove seats by working with your account executive, then assign your purchased seats in the Docker Build Cloud dashboard.
+
+## Add seats
+
+1. Sign in to your [Docker account](https://login.docker.com/) and go to [Docker Build Cloud](https://build.docker.com/).
+2. Select **Settings & billing**.
+3. Under your current plan, select **Add seats**.
+4. Select the number of seats you want to add, then select **Add seats**.
+
+The number of seats will be charged to your payment information on file, and are added immediately.
+
+## Remove seats
+
+1. Sign in to your [Docker account](https:/hub.docker.com/login) and go to [Docker Build Cloud](https://build.docker.com/).
```

Comment:
The other step linked to login.docker.com, this one links to hub.docker.com/login

Reply:
Ah that slipped in it's the wrong URL. But anyway I'll update these to just sign in to Docker Build Cloud!